.stylelintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
We should have a stylelint nextcloud config, that would be nice xD

Reply:
Insert "Well now I'm not doing it" meme here